### ISSUE TITLE
Strokes: Select a more helpful loop after stroke

### ISSUE THREAD

**Issue by jlampel** [CGCookie/retopoflow#1378]
### i-strips
After creating an i stroke on a closed loop, the selection is the loop that was created under the stroke. 

![image](https://github.com/user-attachments/assets/8efc3ace-be9c-4b7e-bcf8-597c50017860)

It would be helpful to instead select the outer loop going around the ring in this case, so that we can continue extruding right away. 

![image](https://github.com/user-attachments/assets/ee262729-c7f4-4c2d-a51e-99bac4b97de4)

### T-strips
There's not much we can do with an inner loop selected, so a similar idea could be applied to T-strips. Instead of this after a stroke from a single selected top loop downwards:

![image](https://github.com/user-attachments/assets/44b5f837-0699-4641-b8f3-ee06bf70c280)

![image](https://github.com/user-attachments/assets/754a5d24-52c9-49b6-908b-8f6b87d27275)

It would be helpful to have the loop that's parallel to the starting loop and at the very end of the stroke selected:

![image](https://github.com/user-attachments/assets/20a699db-eb57-4e94-9d3f-e9febfde4843)

### L-strips
L-strips are a bit trickier, but could still follow similar logic. Instead of the initial selection staying the same after a stroke from the top right vertex to the bottom left vertex:

![image](https://github.com/user-attachments/assets/a26a8da4-80a3-4ce8-9492-7376ae67b9de)

The selection could be the side of the L that is parallel to the original selection:

![image](https://github.com/user-attachments/assets/00eed7bb-1c81-4ff7-9121-7ed5d2888cec)

### C-strips
Same thing really for C-strips. Instead of leaving the current selection:

![image](https://github.com/user-attachments/assets/5da23a79-e4b1-4cd1-9a27-86d89cc72173)

The selection would shift down to the end of the extrusion that's parallel to the starting stroke. 

![image](https://github.com/user-attachments/assets/bb390d2b-4485-41ba-b979-be12fecdbb8a)

### i-bridge
In the case of bridging two strips with an i shape (not sure what exactly to call this one), the user can't extend the stroke any further anyway and we could clear the selection entirely. 

Start: 

![image](https://github.com/user-attachments/assets/620dc1ed-6c68-42dd-abcc-8f11712e2015)

Current result:

![image](https://github.com/user-attachments/assets/8cd18d2f-3b7b-4d73-a168-7f692bffeca0)

Desired result:

![image](https://github.com/user-attachments/assets/aca9d142-6245-4607-a6ca-a2228b7b33cd)

### H-bridge 
Also not sure what to call this one exactly, but if we start with a selection on one side that has two other sides of the patch already defined:

![image](https://github.com/user-attachments/assets/52420663-4976-4936-b25b-d2988010b58d)

Creating a stroke between the two deselected vertices at the end (not yet supported in v4) should select the edges on the boundary that make up the stroke:

![image](https://github.com/user-attachments/assets/d2d9c35c-fa5f-43b2-a0d0-5dfa827001ef)

If going in the other direction, and one of the top sides is selected:

![image](https://github.com/user-attachments/assets/b37b85d2-ca4c-43e1-b4d4-08f3c319d95b)

And the same move is performed, either the boundary of the newly created geo could be selected, similar to above, OR nothing could be selected since the end of the stroke that's parallel to the original selection is not a boundary. Either would be totally reasonable. 

### TLDR 

There's not really any reason to leave inner verts or edges selected, so it's better to only select the boundary edges. Which set of boundary edges? The one parallel to the original selection.

**Comment by vxlcoder** [comment]
outer or inner?

or do similar to polypen and select both inner and outer but bridge only to nearer selection?

**Comment by jlampel** [comment]
Outer in the case above, but inner if the stroke had been drawn in the other direction towards the center of the circle. I was thinking it would be the loop that's attached to the end of the stroke and not the one that the stroke started on.

**Comment by jlampel** [comment]
☝️ Updated the original task with examples of the other shapes.

**Comment by jlampel** [comment]
Works really well and helps to not break the flow when working. Thanks!
